packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -732,10 +749,17 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         return this.channel.applyStashedOp(innerContents.content);
     }
 
-    private verifyNotClosed() {
+    private verifyNotClosed(methodName: string, checkTombstone = true) {
         if (this._disposed) {
             throw new Error("Context is closed");
         }
+
+        if (checkTombstone && this.tombstoned) {
+            throw new DataCorruptionError("Context is tombstoned!", {
+                methodName,
+                contextId: this.id,
```

Comment:
AFAIK, this is PII and we cannot log it. You can tag it as TelemetryDataTag.CodeArtifact and then log it. @markfields to take a look.

Reply:
Removed